### PR TITLE
Update stats borrowing rates

### PR DESCRIPTION
**Pull request by onemicky**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208865803017770

### CODE REVIEW

File: src/domain/synthetics/markets/useMarketsInfoRequest/buildMarketsConfigsRequest.ts [hunk 1]
```diff
@@ -220,6 +220,30 @@ export async function buildMarketsConfigsRequest(
           methodName: "getBytes32",
           params: [prebuiltHashedKeys.virtualLongTokenId],
         },
+        optimalUsageFactorLong: {
```
Comment (divhead) [inline src/domain/synthetics/markets/useMarketsInfoRequest/buildMarketsConfigsRequest.ts:223]: if we need these values only on stats page, let's move it to separated request please, we already have too many of calls for markets, plan to separate it to several requests in the future

**Comment by divhead** [comment]
the base branch need to be changed

**Review by divhead — CHANGES_REQUESTED** [review]
left some comments

File: src/domain/synthetics/markets/useMarketsInfoRequest/buildMarketsConfigsRequest.ts (hunk 1 above)
Comment (onemicky) [reply src/domain/synthetics/markets/useMarketsInfoRequest/buildMarketsConfigsRequest.ts]: Good suggestion! Moved this keys to a separate prebuilt file and separate request.

**Review by divhead — DISMISSED** [review]

**Review by divhead — APPROVED** [review]

**Pull request merged**